### PR TITLE
WIP chore: add descriptions to cli commands

### LINKED ISSUE #1647: better CLI docs?
`pnpm` / `pnpm --help` gives only bare-bones switch listing, can a few extra descriptive words after each be added the the CLI output please?

Also, `pnpm link --help` (etc) with a longer description could help, but perhaps this should be covered by [tldr CLI help system](https://tldr.sh/).

For that matter, the website's docs are a bit sparse.  I'm not an expert at `npm`, but the docs seems to assume users are.  (Perhaps a separate issue, but I was going to copy/paste short descriptions from the website into the CLI. & submit a PR myself, but seems I won't have enough info.)

Thanks for your great tool!

### PR DESCRIPTION
**Pull request by rajasegar**
Fixes #1647

### CODE REVIEW

File: packages/pnpm/src/cmd/help.ts [hunk 1]
```diff
@@ -23,39 +23,49 @@ function getHelpText () {
 
         list: [
           {
+            description: 'Install all dependencies for a project.',
```
Comment (zkochan) [inline packages/pnpm/src/cmd/help.ts]: our current convention is no dot when a single sentence is used. And dot if many sentences are used.

File: packages/pnpm/test/help.spec.ts [hunk 2]
```diff
@@ -8,3 +8,11 @@ test('print an error when help not found', (t) => {
   )
   t.end()
 })
+
+test('print help', (t) => {
+  t.equal(
+    createHelp({})({}, ['foo']).split('\n')[1],
+    '',
```
Comment (zkochan) [inline packages/pnpm/test/help.spec.ts]: what is this test doing?

Comment (rajasegar) [reply packages/pnpm/test/help.spec.ts]: This was supposed to test the newly added command descriptions. I am trying to figure out how to selectively run this test only , can you help me?

File: packages/pnpm/src/cmd/help.ts (hunk 1 above)
Comment (rajasegar) [reply packages/pnpm/src/cmd/help.ts]: This has been fixed in the latest commit.

File: packages/pnpm/test/help.spec.ts (hunk 2 above)
Comment (zkochan) [reply packages/pnpm/test/help.spec.ts]: you can use `test.only(`

**Pull request merged**